content/en/blog/2021-08-15-seccomp-default.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: blog
+title: "Hardening Kubernetes security by enabling seccomp by default"
+date: 2021-08-15
+slug: seccomp-default
+---
+
+**Author:** Sascha Grunert, Red Hat
+
+_Editor's note: Sascha is member of [SIG Release][sigr] and is working on many
+other different container related topics. Feel free to reach him out on Twitter
+[@saschagrunert][twitter] or via the [Kubernetes slack @sascha][slack]._
+
+[sigr]: https://github.com/kubernetes/sig-release
+[twitter]: https://twitter.com/saschagrunert
+[slack]: https://kubernetes.slack.com/team/U53SUDBD4
```

Comment:
We usually put contact details for SIGs rather than for blog authors.

Reply:
Fixed and linked to the SIG Node slack.